### ISSUE TITLE
Request: circling stars icon for "knocked out" or "stunned" effects

### ISSUE THREAD

**Issue by rmunn** [game-icons/icons#790]
There are already two icons that could be used to represent "knocked out" or "stunned" effects — https://game-icons.net/1x1/lorc/stoned-skull.html and https://game-icons.net/1x1/skoll/knockout.html — but I'd like to request one more. What I'd like to see is an icon that represents the "circling stars" look used in many cartoons, with stars (generally between three and seven stars) orbiting the character's head — but with no head in the picture, just a single circle with stars placed on it.

In other words, something vaguely like the circle of stars found in [this image](https://www.dreamstime.com/royalty-free-stock-image-money-knockout-image15965646), but just the circle and stars, no head or other figure. https://game-icons.net/1x1/skoll/knockout.html is very close to what I want, but I'd like to see a simple circle (well, really an oval once you consider perspective) with stars placed along the oval in classic cartoon fashion.

**Comment by Delapouite** [comment]
Here come the stars: https://game-icons.net/1x1/delapouite/knocked-out-stars.html

**Comment by rmunn** [comment]
Terrific, thanks!
